src/layouts/EditHomepage/EditHomepage.jsx

```diff
@@ -820,6 +896,25 @@ const EditHomepage = ({ match }) => {
               )
             }
             setHomepageState(intermediateHomepageState)
+          } else if (val.infocols) {
+            // Create 3 infoboxes by default
+
+            // his happens within the create handler. so at the point of creation
+            // of a new block, this is being added to the last index of the sections array
+            const parentId =
+              updatedHomepageState.frontMatter.sections.length - 1
+            let intermediateHomepageState = updatedHomepageState
+            for (let i = 0; i < 3; i += 1) {
```

Comment:
I am quite confused about this... if we only have 3, should we use this as a constant instead?

Reply:
not always 3. if u see the function where this is being run, is the create operation. When a user creates a infocol section, we by default create 3 infoboxes within it

Reply:
Would prefer this to be a func abstracted out and not in this chunky file similar to what `getDefaultAnnouncementSection` does. 
could we at least define a `DEFAULT_NUMBER_OF_INFO_COLS` for this ya?

Reply:
done